torch/jit/_serialization.py

```diff
@@ -179,7 +179,7 @@ def validate_map_location(map_location=None):
         )
 
     if str(map_location).startswith("cuda"):
-        validate_cuda_device(map_location)
+        validate_device(map_location)
```

Comment:
Should this call be applied to other backends as well? If yes, should this be limited to `['cuda', 'hpu', 'xpu']` (I am not sure that this function will work properly for other backends).

Reply:
I exposed `validate_cuda_device` for backward compatibilty as a tiny wrapper on top of actual `_validate_device`. This change got reverted to original version. Question still remains however whether this call should be generalized to other backends.